engine/Default/planet_ownership.php

```diff
@@ -15,4 +15,10 @@
 $template->assign('Planet', $planet);
 $template->assign('Player', $player);
 
+// Check if this player already owns a planet
+$db->query('SELECT sector_id FROM planet WHERE game_id=' . $db->escapeNumber($player->getGameID()) . ' AND owner_id=' . $db->escapeNumber($player->getAccountID()));
```

Comment:
It might be worth moving this to `$player->getPlanet()`/`$player->hasPlanet()` as it seems like something which would be used elsewhere? (although maybe it isn't, as I guess most places where ownership comes into play is in the planet lists, or from planet -> owner, not owner -> planet)

Reply:
Good idea. Since I added a `SmrPlayer::getPlanet` method in another PR, I'll make a note to come back to this and use it here once that other PR is merged.